src/normalize.ts

```diff
@@ -19,12 +19,25 @@ export const normalizeCodeToPosix = (code: string): string => {
   );
 };
 
+export function normalizeWin32RelativePath(p: string): string {
+  return p.replace(
+    /(['"`])(\.\.([\\/]))+([\w-]+\3)+[^\\/]*\1/g,
```

Comment:
Oh, this was just my temporary workaround for that issue, it didn't cover one level sibling relative path: `./`.